manifests/cinder-csi-plugin/cinder-csi-controllerplugin-rbac.yaml

```diff
@@ -26,6 +26,9 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["csinodes"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["volumeattachments/status"]
+    verbs: ["patch"]
```

Comment:
could we remove nodes at L21 , its not required anymore I suppose.

Reply:
yes, looks like nodes removed from here
https://github.com/kubernetes-csi/external-attacher/blob/master/deploy/kubernetes/rbac.yaml